### ISSUE TITLE
Uploads are disabled.

### ISSUE THREAD

**Issue by weedeater64** [LukeSmithxyz/based.cooking#733]
Go into /src or /data/pix and try to upload and get "Uploads are disabled" error.

**Comment by weedeater64** [comment]
Is this project dead and forgotten?

**Comment by tfasano1** [comment]
Even thought this is a public repository, you are not the owner. Therefore, you cannot make changes directly. I'd recommend learning how to fork and make a pull request. Here's a tutorial https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/

**Comment by weedeater64** [comment]
> Even though this is a public repository, you are not the owner. Therefore, you cannot make changes directly. We wouldn't want random people changing recipes without permission. Of course uploads are disabled!
> 
> I'd recommend learning how to fork and make a pull request. Here's a tutorial https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/

No, I'm not the owner. But the owner literally said in his video to come here and submit recipes by sticking them in the appropriate folders.

https://www.youtube.com/watch?v=ykNEkiYr0QM

He also stated this was all in an effort to de-bloat. How is installing a bunch of software I have no need/want of, then downloading this entire repo just to submit a single recipe de-bloating? 

What you are suggesting is the opposite of de-bloating.

**Comment by hallzy** [comment]
The only way to submit recipes is with a pull request as @tfasano1 mentioned.

In order to do that you have to fork this repo, make changes to your fork, and submit a pull request. The article that @tfasano1 provided looks quite good and is exactly what you need.

If you don't want to use the command line tools mentioned in that article, you can also make changes to your fork on github directly...

Basically:

- click `Fork` in the top right corner of this page
- On your fork do whatever it is you tried to do originally, and it should work this time
-  When you are done with your changes come back to this repo, and click on "Pull Requests", and make a new pull request from your fork.

**Comment by peepopoggers** [comment]
Tell me you have never used github before without telling me you have not used github before :smile_cat:

**Comment by hallzy** [comment]
@weedeater64

just going to ignore that last comment from peep... apart from that comment, if this has given you a solution can you close this issue?
